proto/kava/precisebank/v1/query.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package kava.precisebank.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/kava-labs/kava/x/precisebank/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// Query defines the gRPC querier service for precisebank module
+service Query {
```

Comment:
may be worth adding an endpoint for fractional balance by address as well? i could see it being useful in the event of needing to debug a tx or audit an address.

not a blocking addition

Reply:
yup seems useful and can reduce the headache if we do try debugging it without the query 😄 will add fractional balance & remainder query as well in a followup pr